### PR TITLE
update flightgear to 2020.3.8

### PR DESCRIPTION
**Pull request by violetmage**

### CODE REVIEW

**Comment by violetmage** [comment]
404 >.<

`url-not-found         : <screenshot> failed to download (HTTP 404: Not Found) [http://wiki.flightgear.org/images/3/37/SOTY-2017.jpeg]`

**Comment by Vanadiae** [comment]
I'll have a look later today, or during the week if I can't manage today.

**Comment by Vanadiae** [comment]
> 404 >.<
> 
> `url-not-found : <screenshot> failed to download (HTTP 404: Not Found) [http://wiki.flightgear.org/images/3/37/SOTY-2017.jpeg]`

Yeah, just drop this screenshot for now. I'll add a bunch of nice screenshots from https://wiki.flightgear.org/Category:Screenshot_of_the_month in an other PR.

File: org.flightgear.FlightGear.yaml [hunk 1]
```diff
@@ -105,8 +105,8 @@ modules:
   buildsystem: simple
   sources:
   - type: archive
-    url: https://downloads.sourceforge.net/project/flightgear/release-2020.3/FlightGear-2020.3.6-data.tar.bz2
-    sha256: c11a6c6dd596a82dd9bf59326c2ef7fb971ba79a236b4036db633bb6721a41ff
+    url: https://downloads.sourceforge.net/project/flightgear/release-2020.3/FlightGear-2020.3.8-data.txz
+    sha1: 8f919b22422c8ba708bba22c638416b9878031cc
```
Comment (Vanadiae) [inline org.flightgear.FlightGear.yaml:109]: Please prefer using a sha256 as sha1 is unsafe. You can firs try replacing this line with `sha256: some_foobar_garbage` and the build will give you the correct sha256 saying something like `Expected sha256 some_foobar_garbage but got c11a6c6dd596a82dd9bf59326c2ef7fb971ba79a236b4036db633bb6721a41ff` so here you'll just need to copy paste the sha256 it showed.

**Review by Vanadiae — CHANGES_REQUESTED** [review]
Once the 404 error'ed screenshot is removed and a new build is available, I'll test flightgear then merge it if everything's fine.

**Comment by Vanadiae** [comment]
So I played a bit and I didn't notice any problem, so I'm merging :) Thanks again for updating flightgear! I noticed that sourceforge provides RSS feeds for releases files, so I'm adding it to my feed reader so hopefully next time I'll notice there's an update available and so, update the flatpak here :)

**Comment by Vanadiae** [comment]
BTW your git commit email is not set correctly. It seems like it's your computer's hostname instead?

**Comment by violetmage** [comment]
ah, is true

many thanks for information :)

**Pull request merged**